scripts/builtin/steplm.dml

```diff
@@ -0,0 +1,320 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please put a "Modifications Copyright 2019 Graz University of Technology" similar to the lmCG builtin function on top.